__test__/app.test.js

```diff
@@ -61,7 +62,27 @@ describe('/api', () => {
             food_item_id: 1,
             name: 'New York Cheesecake',
             price: 4.5,
-            course: 'dessert'
+            course: 'dessert',
+            is_active: true,
+          });
+        });
+    });
+    it('SUCCESS status 201 - changes active state of a food item', () => {
```

Comment:
I was told on backend nc news that PATCH is a 200 status code when successful

Reply:
change that to 200